qlib/data/data.py

```diff
@@ -66,7 +64,14 @@ def backend_obj(self, **kwargs):
             provider_uri_map = backend_kwargs.setdefault("provider_uri_map", {})
             freq = kwargs.get("freq", "day")
             if freq not in provider_uri_map:
-                provider_uri_map[freq] = C.dpm.get_data_path(freq)
+                try:
```

Comment:
We have two method named get_data_path
![image](https://user-images.githubusercontent.com/465606/133203215-46379089-0812-4383-b625-faf979efaf46.png)